webserver/static/robots.txt

```diff
@@ -2,3 +2,5 @@ User-agent: *
 Disallow: /current-status
 Disallow: /user/
 Disallow: /1/
+Disallow: /api
```

Comment:
There's a file /api.py which needs to be blocked for bots . /api means every file will be blocked with '/api' where '/api/' will block /api/ folder (which is not present in LB) not a file :)